problem1.swift

```diff
@@ -2,4 +2,26 @@
 
 let input1 = Int(readLine()!)!
 let input2 = Int(readLine()!)!
+var i = input1
+var sum = 0
+
+if input1 < input2 { 
+    while i <= input2 { 
+        sum = sum + i
+        i = i + 1
+    }
+} 
```

Comment:
Normally the only whitespace between the if statement's end curly brace and the else or else if is a space. Following conventions will make your code easier to read and edit.